fe/fe-core/src/main/java/com/starrocks/server/StorageVolumeMgr.java

```diff
@@ -164,6 +165,14 @@ public void updateStorageVolume(String name, Map<String, String> params, Optiona
             StorageVolume copied = new StorageVolume(sv);
             validateParams(copied.getType(), params);
 
+            List<String> immutableProperties =
```

Comment:
How about making `immutableProperties` as a member of `StorageVolumeMgr`?

Reply:
yes it can. However, I am thinking about refactor the StorageVolume, derive it by S3StorageVolume/HdfsStorageVolume/AzblobStorageVolume, each storage volume can handle its own immutable properties and validation.

It's beyond this feature, so in this PR, just keep the simplest coding, refactor it in a separate PR.